### PR TITLE
TIP-842: Move attritbute option classes to Structure

### PR DESCRIPTION
**Pull request by arnolanglade**
**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added legacy Behats               | -
| Added acceptance tests            | -
| Added integration tests           | -
| Changelog updated                 | yes
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | -
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: src/Pim/Bundle/EnrichBundle/Form/Type/AttributeOptionCreateType.php [hunk 1]
```diff
@@ -33,18 +33,18 @@ public function __construct($dataClass)
...
-                'optionValues',
-                CollectionType::class,
-                [
-                    'type'         => AttributeOptionValueType::class,
-                    'allow_add'    => true,
-                    'allow_delete' => true,
-                    'by_reference' => false
-                ]
-            );
+//        $builder
```
Comment (arnolanglade) [inline src/Pim/Bundle/EnrichBundle/Form/Type/AttributeOptionCreateType.php]: to revert

File: UPGRADE.md [hunk 2]
```diff
@@ -247,4 +247,5 @@ find ./src/ -type f -print0 | xargs -0 sed -i 's/Pim\\Component\\Catalog\\Reposi
 find ./src/ -type f -print0 | xargs -0 sed -i 's/Pim\\Bundle\\EnrichBundle\\Controller\\Rest\\GroupTypeController/Akeneo\\Pim\\Structure\\Bundle\\Controller\\InternalApi\\GroupTypeController/g'
 find ./src/ -type f -print0 | xargs -0 sed -i 's/Pim\\Component\\Connector\\ArrayConverter\\FlatToStandard\\GroupType/Akeneo\\Pim\\Structure\\Component\\ArrayConverter\\FlatToStandard\\GroupType/g'
 find ./src/ -type f -print0 | xargs -0 sed -i 's/Pim\\Component\\Connector\\ArrayConverter\\StandardToFlat\\GroupType/Akeneo\\Pim\\Structure\\Component\\ArrayConverter\\StandardToFlat\\GroupType/g'
-find ./src/ -type f -print0 | xargs -0 sed -i 's/Pim\\Component\\Catalog\\Repository\\GroupTypeRepositoryInterface/Akeneo\\Pim\\Structure\\Component\\Repository\\GroupTypeRepositoryInterface/g'
```
Comment (jjanvier) [inline UPGRADE.md]: why do you remove it?

File: src/Akeneo/Pim/Structure/Bundle/Form/Type/AttributeOptionCreateType.php [hunk 3]
```diff
@@ -1,7 +1,8 @@
 <?php
 
-namespace Pim\Bundle\EnrichBundle\Form\Type;
+namespace Akeneo\Pim\Structure\Bundle\Form\Type;
 
+use Pim\Bundle\EnrichBundle\Form\Type\AttributeOptionValueType;
```
Comment (jjanvier) [inline src/Akeneo/Pim/Structure/Bundle/Form/Type/AttributeOptionCreateType.php]: why don't you move it?

File: UPGRADE.md (hunk 2 above)
Comment (willy-ahva) [reply UPGRADE.md]: Yes why ?

File: src/Akeneo/Pim/Structure/Bundle/Form/Type/AttributeOptionCreateType.php (hunk 3 above)
Comment (willy-ahva) [reply src/Akeneo/Pim/Structure/Bundle/Form/Type/AttributeOptionCreateType.php]: Yes why ?

Comment (arnolanglade) [reply src/Akeneo/Pim/Structure/Bundle/Form/Type/AttributeOptionCreateType.php]: See IRL !

**Pull request merged**